scss/components/_posts.scss

```diff
@@ -2,6 +2,7 @@
 
 .post-feed {
   border-top: 1px solid $gray;
+  border-bottom: 1px solid $gray;
   padding: u(1rem 0 1rem 2rem);
```

Comment:
I made a note in https://github.com/18F/openFEC-web-app/pull/1731 about the `padding-left` indent. Just want to make sure if it's intentional or needs a fix.